src/util_misc.cpp

```diff
@@ -386,7 +386,10 @@ int Util::readProcMapsLine(int mapsfd, ProcMapsArea *area)
   c = readDec (mapsfd, (VA *)&inodenum);
   area -> name[0] = '\0';
   while (c == ' ') c = readChar (mapsfd);
-  if (c == '/' || c == '[') { /* absolute pathname, or [stack], [vdso], etc. */
+  if (c == '/' || c == '[' || c == '(') {
+    // absolute pathname, or [stack], [vdso], etc.
+    // On some machines, deleted files have a " (deleted)" prefix to the
+    // filename.
```

Comment:
If `c == '('` is checking for the pattern "(deleted)" as a prefix, then let's have it really check for that entire string.  The kernel developers could later introduce something else like "(optimized)" or some other weird string.  Since there is no standard operating here, we need to protect ourselves, to be future-proof.